pennylane/tape/measure.py

```diff
@@ -72,16 +72,24 @@ def __init__(self, return_type, obs=None, wires=None, eigvals=None):
         # Below, we imitate an identity observable, so that the
         # device undertakes no action upon recieving this observable.
         self.name = "Identity"
-        self.diagonalizing_gates = lambda: []
```

Comment:
A bit surprised this changes?

Reply:
This was just so that the `TestCircuitGraphDiagonalizingGates` test passes 🤔 

Alternatively, I could just delete that test class?

Reply:
I didn't know the circuit graph has functionality for diagonalisation in the first place...